lib/debezium/types.go

```diff
@@ -134,12 +134,21 @@ func (f Field) ParseValue(value any) (any, error) {
 		DateKafkaConnect,
 		TimeKafkaConnect,
 		DateTimeKafkaConnect:
-		if _, ok := value.(float64); !ok {
-			// Since this value is coming from Kafka, and will have been marshaled to a JSON string, it should always
-			// be a float64. Let's check this if this assumption holds and if so clean up the code below so that we
-			// aren't doing float -> string -> float.
+
+		switch value.(type) {
+		case float64:
+			// This value is coming from Kafka, and will have been marshaled to a JSON string, so when it is
+			// unmarshalled it will be a float64
+			// -> Pass.
+		case int64:
```

Comment:
Aren't we marshaling and unmarshaling from Reader before calling Transfer? Why is this happening?

Reply:
We are, I want to see if we can avoid having to do so since marshaling and unmarshaling are performance heavy operations and will reduce throughput.